### DISCUSSION TITLE
Global Request Headers

### DISCUSSION THREAD

**Discussion by runoncedev** [Huachao/vscode-restclient#1181] [Ideas]
Right now I'm using file variables to set the `content-type` and the `return-type` of all my requests:

```
@contentType = application/json

###

{{baseUrl}}/foo
content-type: {{contentType}}
accept: {{contentType}}
```

I think it would be very helpful if instead we could define some sort of "global headers" so we don't need to repeat the exact same headers for every request.

**Comment by datenimperator** [comment]
That was my idea exactly. I need to include the same Authorization header into every request which isn't DRY. Any chance to improve that?
